### PR TITLE
Don't run bundle update in the Dockerfile

### PR DESCRIPTION
**Pull request by mstanleyjones**
This avoids the US-ASCII bug in jekyll-redirect plugin

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -1,5 +1,5 @@
 FROM starefossen/github-pages:137
 
-COPY . /usr/src/app/
+COPY . /usr/src/app
```
Comment (thaJeztah) [inline Dockerfile]: Can you change this back, as it doesn't seem related to this change

Comment (mstanleyjones) [inline Dockerfile]: It is related. One is what @johndmulhausen told me to use and the other one (without the `/`) is what is in the `onbuild` version of the `github-pages` plugin. Probably it doesn't matter but I thought maybe it was best to stick to how they had it.
